src/main/java/frc/robot/subsystems/Intake/BetaIntake.java

```diff
@@ -0,0 +1,118 @@
+package frc.robot.subsystems.Intake;
+
+import java.util.function.BooleanSupplier;
+
+
+import frc.util.Motor.MotorHelper;
+import frc.util.Sensors.absoluteEncoder.AbsoluteEncoder;
+import frc.util.SmartDashboard.PIDTuning;
+import frc.util.SmartDashboard.TunableNumber;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+
+import com.revrobotics.spark.SparkBase.ControlType;
+import com.revrobotics.spark.ClosedLoopSlot;
+import com.revrobotics.spark.SparkMax;
+
+public class BetaIntake extends Intake {
```

Comment:
please remove the singleton pattern from this subsystem. also you don't need to do the thing where you make an intake folder and an intake interface and so on and so forth. You can just put the intake state enum in either the intake file or in constants